code/game/objects/items/weapons/storage/fancy.dm

```diff
@@ -197,6 +198,62 @@
 	. = ..()
 	fill_cigarre_package(src, list("fuel" = 15))
 
+/obj/item/weapon/storage/fancy/cigarettes/homeless
+	name = "\improper Nomads packet"
+	desc = "A packet of six Nomads cigarettes. Nomads's Extra strong for when your life is more extra hard"
+	icon_state = "Cpacket"
+	item_state = "Cpacket"
+
+/obj/item/weapon/storage/fancy/cigarettes/homeless/Initialize()
+	. = ..()
+	fill_cigarre_package(src, list("adrenaline" = 6))
+
+/obj/item/weapon/storage/fancy/cigcartons
+	name = "carton of cigarettes"
+	desc = "A box containing 10 packets of cigarettes."
+	icon_state = "cigpacketcarton"
+	item_state = "cigpacketcarton"
+	icon = 'icons/obj/cigarettes.dmi'
+	w_class = ITEM_SIZE_NORMAL
+	throwforce = WEAPON_FORCE_HARMLESS
+	storage_slots = 10
+	item_obj = /obj/item/weapon/storage/fancy/cigarettes
+	can_hold = list(/obj/item/weapon/storage/fancy/cigarettes)
+	icon_type = "packet"
+	reagent_flags = REFILLABLE | NO_REACT
+
+/obj/item/weapon/storage/fancy/cigcartons/update_icon()
+	if( contents.len > 0 )
+		icon_state = "[initial(icon_state)]1"
+	else
+		icon_state = "[initial(icon_state)]"
+
+/obj/item/weapon/storage/fancy/cigcartons/populate_contents()
+	for(var/i in 1 to storage_slots)
+		new item_obj(src)
+	update_icon()
+
+/obj/item/weapon/storage/fancy/cigcartons/dromedaryco
+	name = "carton of dromedaryco cigarettes"
+	desc = "A box containing 10 packets of dromedarycos cigarettes."
```

Comment:
The other company names are capitalized. Why this one isn't?

Reply:
Copy paste straight out of path name

Reply:
Should still be capitalized. This is the name of a company isn't it?